pkgs/tools/inputmethods/fcitx5/default.nix

```diff
@@ -20,7 +20,6 @@
 , enchant
 , gdk-pixbuf
 , libGL
-, libuv
```

Comment:
I am opposite to removing libuv because fcitx5 can be used with nix on non-nixos system which based on openrc or something else, though those cases are uncommon.

Reply:
But as long as sd-event is available, libuv is no searched/linked to the output. So this has always been a nop.

Reply:
If systemd is found, libuv is never used and systemd is in dependencies unconditional currently. In this case, let us remove libuv.